Dockerfile

```diff
@@ -1,37 +1,173 @@
-FROM alpine:3.12
+FROM fedora:rawhide AS builder
 
-# https://github.com/opencontainers/image-spec/blob/master/annotations.md
 ARG BUILD_DATE
 ARG VCS_REF
+ARG MARIADB_VERSION="10.4.15"
```

Comment:
fyi: this will be passed as arg, not set statically. This is just me being lazy.